app/controllers/api/v0/articles_controller.rb

```diff
@@ -1,6 +1,24 @@
 class Api::V0::ArticlesController < ApplicationController
+  before_action :authenticate_api_v0_user!, only: [:create]
+  
   def index
     articles = Article.all
     render json: articles, each_serializer: Articles::IndexSerializer
   end
+
+  def create
+    if current_api_v0_user.role === 'research_group'
```

Comment:
please try:

``` 
current_api_v0_user. research_group?
```

Reply:
tried it, and it's working. Is this considered better practice as oposed to the '==='?

Reply:
Yes